commons/all/src/test/java/org/infinispan/commons/time/ControlledTimeService.java

```diff
@@ -32,7 +32,7 @@ public Instant instant() {
       return Instant.ofEpochMilli(currentMillis);
    }
 
-   public void advance(long time) {
+   public synchronized void advance(long time) {
```

Comment:
Do we really have a case that requites this to be synchronized? I can see having it volatile so other threads will for sure see the correct time, but I would think if a test has to protect from this being called concurrently that there is something probably wrong.

Reply:
no reason, it was just to remove the IDEA warning that it is a non-atomic volatile update (I very sensitive settings xD). 
I can remove it if you want.